mcs/build/common/basic-profile-check.cs

```diff
@@ -41,9 +41,9 @@ static int Main ()
 
 		Version min_mono_version;
 #if __MonoCS__
-		min_mono_version = new Version (6, 2);
+		min_mono_version = new Version (6, 5);
```

Comment:
Remove if?

Reply:
Hm this is for cases where CSC needs a newer/different mono than MCS, so I think it's fine to keep it.